toolkit/tools/grapher/grapher.go

```diff
@@ -335,10 +369,14 @@ func populateGraph(graph *pkggraph.PkgGraph, repo *pkgjson.PackageRepo) (err err
 	timestamp.StopEvent(nil) // add package nodes
 	timestamp.StartEvent("add dependencies", nil)
 
+	// Sort the map to ensure the order is deterministic
```

Comment:
For my information: this has (to your knowledge) no functional impact and you're doing this to make debugging and easier and runs more deterministic, right? In other words: things should still build even if we didn't sort the packages.

Reply:
Correct, this shouldn't matter functionally at all. Maps are always non-deterministically ordered so 'sorted' order is just a subset of those.